web/profiles/joinup/config/install/image.style.profile_icon.yml

```diff
@@ -0,0 +1,13 @@
+langcode: en
+status: true
+dependencies: {  }
+name: profile_icon
+label: 'Profile icon'
+effects:
+  d5edf7b7-b996-4fae-a966-9908a9adb054:
+    uuid: d5edf7b7-b996-4fae-a966-9908a9adb054
```

Comment:
This is exported when running drush cde, so even when removing is possible, it would get re-added each time you export...